### PR TITLE
Fixing profile listing

### LINKED ISSUE #2727: checker profiles cannot be listed does not list the available profiles
**Describe the bug**
CodeChecker checkers --profile list 
does not list the available profiles. Returns with an empty list.

In earlier versions it returned this:
/6.11.1/CodeChecker/bin/CodeChecker checkers --profile list
 default
 sensitive
 security
 portability
 extreme

**CodeChecker version**
CodeChecker analyzer version:
---------------------------------------------------------------
Kind                 | Version
---------------------------------------------------------------
Base package version | 6.12.0
Package build date   | 2020-05-11T22:04
Git commit ID (hash) | 24c0414c8f5250ce81534d43d9cf64b9b07a430e
Git tag information  | 6.12
---------------------------------------------------------------

CodeChecker web version:
------------------------------------------------------------------------
Kind                          | Version
------------------------------------------------------------------------
Base package version          | 6.12.0
Package build date            | 2020-05-11T22:04
Git commit ID (hash)          | 24c0414c8f5250ce81534d43d9cf64b9b07a430e
Git tag information           | 6.12
Server supported API (Thrift) | 6.26
Client API (Thrift)           | 6.26

### PR DESCRIPTION
**Pull request by dkrupp**
Fixes #2727

### CODE REVIEW

File: analyzer/codechecker_analyzer/cmd/checkers.py [hunk 1]
```diff
@@ -209,7 +209,9 @@ def uglify(text):
             rows = context.available_profiles.items()
         else:
             header = ['Profile name']
-            rows = ([key] for key in context.available_profiles.keys())
+            rows=[]
```
Comment (csordasmarton) [inline analyzer/codechecker_analyzer/cmd/checkers.py:212]: You can insert a tuple in the short array creation syntax the following way:
```suggestion
            rows = ([(key, "")] for key in context.available_profiles.keys())
```

Comment (dkrupp) [reply analyzer/codechecker_analyzer/cmd/checkers.py]: I fixed as you suggested + added a test case

File: analyzer/tests/functional/cmdline/test_cmdline.py [hunk 2]
```diff
@@ -88,6 +88,14 @@ def test_checkers(self):
         checkers_cmd = [env.codechecker_cmd(), 'checkers']
         self.assertEqual(0, run_cmd(checkers_cmd)[0])
 
+    def test_checkers_profile(self):
+        """ Listing available checkers. """
```
Comment (gyorb) [inline analyzer/tests/functional/cmdline/test_cmdline.py:92]:
```suggestion
        """ Listing available checker profiles. """
```

**Comment by gyorb** [comment]
Please check travis too.

File: analyzer/tests/functional/cmdline/test_cmdline.py (hunk 2 above)
Comment (dkrupp) [reply analyzer/tests/functional/cmdline/test_cmdline.py]: fixed

**Pull request merged**